wifi_passview_v_0.2.bat

```diff
@@ -0,0 +1,84 @@
+@echo off
+color b
+echo Administrative permissions required. Detecting permissions.
+ping localhost -n 2 >NUL
+cls
+echo Administrative permissions required. Detecting permissions..
+ping localhost -n 2 >NUL
+cls
+echo Administrative permissions required. Detecting permissions...
+ping localhost -n 2 >NUL
```

Comment:
Let's remove this, we don't need to fool people that has a background process happening... we can make it one echo and check for the administrative permission.

Reply:
I just try running this command... and I think this is okay because it is not too long to add delay. I think this is okay now for me.